### ISSUE TITLE
Why do I need to set the relation field twice in 0.22 ?

### ISSUE THREAD

**Issue by bangbaew** [steebchen/prisma-client-go#986]
Normally this code works:
```go
createFullAccess := prisma.Client.Access.CreateOne(
		db.Access.CompanyRelation.Link(db.Company.Path.Equals(i.Path)),
		db.Access.Email.Set(c.Locals("email").(string)),
		db.Access.Read.Set(true),
		db.Access.Write.Set(true),
).Tx()
```

but in 0.22, i have to add one more line
```go
createFullAccess := prisma.Client.Access.CreateOne(
		db.Access.CompanyRelation.Link(db.Company.Path.Equals(i.Path)),
		db.Access.CompanyPath.Set(i.Path), // This is the newly required line
		db.Access.Email.Set(c.Locals("email").(string)),
		db.Access.Read.Set(true),
		db.Access.Write.Set(true),
).Tx()
```

I don't think I need to code 2 lines for each relation field, maybe I'm missing something from the docs?

**Comment by steebchen** [comment]
Hey this is probably a bug, looking into it today. Please revert to the v0.21 for now

**Comment by steebchen** [comment]
If you would be able to share your schema for the `Access` and `Company`, that would be great 🙏

**Comment by bangbaew** [comment]
> If you would be able to share your schema for the `Access` and `Company`, that would be great 🙏

Here they are
```prisma
model company {
    path     String     @id
    name     String
    image    String?
    document document[]
    access   access[]
}

model access {
    company_relation company @relation(fields: [company_path], references: [path])
    company_path     String

    email String
    read  Boolean @default(false)
    write Boolean @default(false)

    @@id([company_path, email])
}
```

**Comment by steebchen** [comment]
Fixed in [v0.22.1](https://github.com/steebchen/prisma-client-go/releases/tag/v0.22.1)
